src/docs/community/innersourcing.mdx

```diff
@@ -0,0 +1,44 @@
+import { Divider, ImageGrid, NewFeatureFormButton } from '../../app/components';
+import FourStepsImg from '../../app/assets/pxblue-conformity.svg';
+
+# Innersource into PX Blue
+
+One of the primary goals of Power Xpert Blue is to eliminate the need for teams to duplicate effort. If you have built something using Power Xpert Blue that you think would benefit other development teams, we would love to hear from you. Please click the button below and fill out the form to submit your new ideas.
+
+<NewFeatureFormButton />
+
+Alternatively, you may [send us an email](/community/contactus).
+
+## Developers
+
+Developers can also choose add a feature by [creating a pull request on our GitHub](https://github.com/pxblue). Please check out our [Contribution Guidelines](https://github.com/pxblue/.github/blob/master/CONTRIBUTING.md) before you get started.
+
+We build our code examples using GitHub and use several online tools for sharing them. These tools allow for importing GitHub repositories and making the code runnable and editable in an online IDE.
+
+-   For Angular and Ionic projects, we use [StackBlitz](http://www.stackblitz.com).
+-   For React we use [Code Sandbox](https://codesandbox.io).
+-   For React Native we use [Snack](https://snack.expo.io/). Snack requires that your project is built using the [Expo CLI](https://docs.expo.io/workflow/expo-cli/).
+
+If you have not used these tools before, they're a great way to play with code and make sharing code snippets much easier. We recommend that you try them out when submitting code samples to speed up the process.
+
+## How We Innersource New Features
+
+<ImageGrid
+    images={[FourStepsImg]}
+    gridImageProps={{ sm: 8 }}
+    caption={'We take four steps to go through a newly proposed idea: evaluate, iterate, implement, and release.'}
+/>
+
+The first step after we hear from you is to evaluate the popularity, importance, and feasibility of the proposed idea. How "trendy" is this practice? How many Eaton applications are using it? How many will be using it in the future? Is this a low-hanging fruit or does it require intense attention? Once this is decided, we will get back to you whether or not we can inner source it and, if we do, what the timeline looks like.
+
+Next, we will gather our designers and developers to iterate for a better PX Blue conformity, if necessary. We would check for colors, components, and layouts to ensure that the proposed idea aligns with our existing patterns. During this iteration process, we might contact you from time to time to exchange ideas.
+
+We then queue it up for a scheduled quarter release (see our [roadmap](/roadmap) for details). Depends on the demand, we would implement it in the Figma team library (designers), NPM packages, and/or design pattern code examples (developers).
+
+Finally, we officially release it. For developers, you might be able to use our pre-released beta packages for a preview.
```

Comment:
```suggestion
```

Reply:
So no mentioning of those pre-release packages?

Reply:
I would shy away from encouraging people to use them in their applications. But if you want to say something about it you could say something like "Occasionally, code may be available for testing in the form of alpha or beta packages prior to its official release"